protobuf.lua

```diff
@@ -0,0 +1,612 @@
+
+    -- -------------------------------------------------------------
+    -- configuration { "androidgles3_x86_debug" }
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_androidgles3_x86_debug.lua")
+
+    -- project specific configuration settings
+
+    -- configuration { "androidgles3_x86_debug" }
+
+    -- -------------------------------------------------------------
+    -- configuration { "androidgles3_x86_release" }
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_androidgles3_x86_release.lua")
+
+    -- project specific configuration settings
+
+    -- configuration { "androidgles3_x86_release" }
+
+    -- -------------------------------------------------------------
+    -- configuration { "androidgles3_armv8_debug" }
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_androidgles3_armv8_debug.lua")
+
+    -- project specific configuration settings
+
+    -- configuration { "androidgles3_armv8_debug" }
+
+    -- -------------------------------------------------------------
+    -- configuration { "androidgles3_armv8_release" }
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_androidgles3_armv8_release.lua")
+
+    -- project specific configuration settings
+
+    -- configuration { "androidgles3_armv8_release" }
+
+    -- -------------------------------------------------------------
+    -- configuration { "androidgles3_x64_debug" }
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_androidgles3_x64_debug.lua")
+
+    -- project specific configuration settings
+
+    -- configuration { "androidgles3_x64_debug" }
+
+    -- -------------------------------------------------------------
+    -- configuration { "androidgles3_x64_release" }
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_androidgles3_x64_release.lua")
+
+    -- project specific configuration settings
+
+    -- configuration { "androidgles3_x64_release" }
+
+    -- -------------------------------------------------------------
+  end
+
+  if (_TARGET_IS_WINUWP) then
+    -- -------------------------------------------------------------
+    -- configuration { "winuwp" } == _TARGET_IS_WINUWP
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_winuwp.lua")
+
+    -- project specific configuration settings
+
+    -- configuration { "windows" }
+
+    defines {
+      "_CRT_SECURE_NO_WARNINGS",
```

Comment:
@mfpdias you should be able to define _CRT_NONSTDC_NO_DEPRECATE in order to build to get rid of the deprecation notice.  We prefer this over changing the 3rdparty library.